src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import { getAllArticles } from "../utils/api";
+import { Link } from 'react-router-dom';
+import Votes from "./Votes";
+
+const ArticlesList = () => {
+    const [isLoading, setIsLoading] = useState(true);
+    const [articles, setArticles] = useState([])
+    
+    useEffect(() => {
+        getAllArticles().then((data) => {
+            setArticles(data)
+            setIsLoading(false);
```

Comment:
good that you are using a loading state.  we would want to to setIsLoading(true) on line 10.5 as well

Reply:
ok, I’ll add it.